storybook/storybook-theme/package.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
I'm not sure we should make a seperate package for this. If we add `@storybook/theming` to both storybooks and just use this directory to store a generic theme config, we could keep it a lot more simple. Or maybe even easier: I think we only need to define the theme in the main storybook (storybook-css), right? Then we could do everything in that directory.

Reply:
I had to add npm to be able to use theming, the alternative is the storybook directories having their own theme config.

Reply:
I can remove main though

Reply:
Maybe we can just store the config in `/storybook` (the object with `brandTitle` and `brandImage`), and create the themes seperately in both storybooks.

Reply:
Changed it to suggested solution

Reply:
One suggestion (last one, I swear ;) ) : you suggest a `/storybook-theme` directory with one file in it. Couldn't we just place a file directly in the `/storybook` directory, and call it `themeConfig.js` or something?